doc/ImplementingApplications/Steps2IntegrateBasicServices/Steps2IntegrateBasicServices.md

```diff
@@ -35,7 +35,7 @@ Now , the basic services implemented in the onf-core-model-ap-bs/basicServices m
   ```
   let responseHeader = await restResponseHeader.createResponseHeader(xCorrelator, startTime, req.url);
   ```
-  A formulated response header will consists of the xCorrelator sent in the request , exec-time and the lifecycle-state of the request url.
+  A formulated response header will consists of the xCorrelator sent in the request , start-time and the lifecycle-state of the request url.
```

Comment:
This change is not required response header consists of exec-time not the start time